src/Field/Field.tsx

```diff
@@ -54,6 +56,13 @@ export const Field = ({
           )}
         </Box>
       )}
+      <Box>
+        {assistiveText && (
+          <Text tag="p" typo="body" color="subtext">
```

Comment:
Also body doesn't exist as part of the new typo. It's eaither `body-regular` and in that case you don't need the `typo` since it's default or `body-small`

Reply:
@Jmuskett Not sure if you saw this comment, did you check margins between the title and assistive text? And also how margin between label/assistive text and input under it?

Reply:
Added in [616b2a](https://github.com/marshmallow-insurance/smores-react/pull/2118/commits/616b2a5e21517f733b6649623a18b1642444226d)